pkg/logs/internal/launchers/file/launcher.go

```diff
@@ -342,10 +342,20 @@ func (s *Launcher) createRotatedTailer(t *tailer.Tailer, file *tailer.File, patt
 }
 
 func CheckProcessTelemetry(stats *util.ProcessFileStats) {
-	if float64(stats.AgentOpenFiles)/float64(stats.OsFileLimit) > 0.9 {
-		log.Warnf("Agent process is close to OS file limit of %v. Agent process currently has %v files open.", stats.OsFileLimit, stats.AgentOpenFiles)
-	} else if stats.AgentOpenFiles/stats.OsFileLimit >= 1 {
-		log.Errorf("Agent process has reached the OS open file limit: %v. This may be preventing log files from being tailed by the Agent and could interfere with the basic functionality of the Agent. OS file limit must be increased.", stats.OsFileLimit)
+	ratio := float64(stats.AgentOpenFiles) / float64(stats.OsFileLimit)
+	if ratio > 0.7 {
+		log.Warnf("Agent process has %v files open (%0.f%%), which is over 70%% of OS open file limit (%v). Consider increasing OS file limit.",
```

Comment:
Maybe a bit clearer to say something like `Agent process has 30 files open which is 75% of the OS open file limit (40). This is over 70% utilization, consider increasing the OS open file limit`.

Right now its not 100% clear what the percentage refers to. Wdyt?